libs/community/langchain_community/chat_models/baichuan.py

```diff
@@ -366,18 +366,15 @@ def lc_serializable(self) -> bool:
     """Maximum number of tokens to generate."""
     request_timeout: int = Field(default=60, alias="timeout")
     """request timeout for chat http requests"""
-    model: str = "Baichuan2-Turbo-192K"
-    """model name of Baichuan, default is `Baichuan2-Turbo-192K`,
+    model: str = "Baichuan2-Turbo"
+    """model name of Baichuan, default is `Baichuan2-Turbo`,
     other options include `Baichuan2-Turbo`"""
     temperature: Optional[float] = Field(default=0.3)
     """What sampling temperature to use."""
     top_k: int = 5
     """What search sampling control to use."""
     top_p: float = 0.85
     """What probability mass to use."""
-    with_search_enhance: bool = False
```

Comment:
why is this deleted? Given that the comment says that it's kept for backwards compatibility?